### PR TITLE
Support OCI manifest lists in image mirror

### PR DESCRIPTION
**Pull request by Jamstah**
We only need to tell the registry we accept them, the registry client code already handles OCI manifests transparently for us.

### CODE REVIEW

File: pkg/cli/image/manifest/manifest.go [hunk 1]
```diff
@@ -214,6 +214,7 @@ type FilterFunc func(*manifestlist.ManifestDescriptor, bool) bool
 // PreferManifestList specifically requests a manifest list first
 var PreferManifestList = distribution.WithManifestMediaTypes([]string{
 	manifestlist.MediaTypeManifestList,
+	imagespecv1.MediaTypeImageIndex,
```
Comment (ardaguclu) [inline pkg/cli/image/manifest/manifest.go:217]: AFAICS, `imagespecv1.MediaTypeImageManifest` was added to support OCI manifests in [here](https://github.com/openshift/oc/pull/697/files#diff-88a19aa652d0dad2300f5b2080f14494546c5036b5b357e07ae150e24d67c072R216), what is the reason of adding this one?

**Review by ardaguclu — COMMENTED** [review]
Thanks for this patch. I have just one question, other than that;

/lgtm

File: pkg/cli/image/manifest/manifest.go (hunk 1 above)
Comment (ardaguclu) [reply pkg/cli/image/manifest/manifest.go:217]: ah, I missed that sorry. You're right.

Comment (Jamstah) [reply pkg/cli/image/manifest/manifest.go:217]: I should have titled it right :)

**Pull request merged**